code/modules/mining/fulton.dm

```diff
@@ -141,14 +137,14 @@ var/list/total_extraction_beacons = list()
 
 /obj/item/fulton_core
 	name = "extraction beacon signaller"
-	desc = "Emits a signal which fulton recovery devices can lock on to. Craft with metal to create a beacon."
+	desc = "Emits a signal which fulton recovery devices can lock on to."
 	icon = 'icons/obj/stock_parts.dmi'
 	icon_state = "subspace_amplifier"
 
 /datum/crafting_recipe/fulton
 	name = "Fulton Recovery Beacon"
 	result = /obj/structure/extraction_point
-	reqs = list(/obj/item/fulton_core = 1, /obj/item/stack/sheet/metal = 5)
```

Comment:
Is this supposed to not have metal for a frame? Seems like that would be a logical step. Would be overly powerful if you could just throw it anywhere